src/scales/scale.logarithmic.js

```diff
@@ -48,11 +48,14 @@ module.exports = function(Chart) {
 						if (valuesPerType[meta.type] === undefined) {
 							valuesPerType[meta.type] = [];
 						}
-
+                        
 						helpers.each(dataset.data, function(rawValue, index) {
 							var values = valuesPerType[meta.type];
 							var value = +me.getRightValue(rawValue);
-							if (isNaN(value) || meta.data[index].hidden) {
+                            if(value === 0){
+                                dataset.data[index] = null;  // don't show y-value
```

Comment:
I agree this may be not an ideal solution. 
The idea is not only exlude 0 from range calculation, but also show on the chart only x-valye in this case (like it is done in excel).
I'll try to found the way of doing it without changing to null. If you can indicate the direction, please comment this.